CallKitDemo/ProviderDelegate.swift

```diff
@@ -200,11 +200,19 @@ final class ProviderDelegate: NSObject, CXProviderDelegate {
         
         // If we are returning from a hold state
         if answerCall?.hasConnected ?? false {
-            configureAudioSession()
+            //configureAudioSession()
+            var userInfo = Dictionary<AnyHashable, Any>()
```

Comment:
I think this is the same code that the if below, I think it would be nice to wrap it into a function